app/assets/stylesheets/common/_markus.scss

```diff
@@ -238,8 +229,8 @@ button,
 /** Fieldset */
 
 fieldset {
-  @include border-radius($radius);
-  @include box-shadow(0 3px 0 $grey);
+  border-radius: $radius;

```

Comment:
Properties should be ordered background, border-radius, box-shadow, padding, position